.github/workflows/run-k8s-tests-on-amd64.yaml

```diff
@@ -34,14 +34,11 @@ jobs:
           - qemu
         container_runtime:
           - containerd
-        snapshotter:
-          - devmapper
```

Comment:
So with the devmapper snapshotter removed, we are both removing a failing test, but presumably adding a test that we didn't have coverage here before (maybe it defaults to overlayfs in the absence of this). Do we already have coverage for this in other tests and can we just test qemu+crio+k0s here?